src/routers/swap_router.ts

```diff
@@ -8,6 +8,7 @@ import { SwapService } from '../services/swap_service';
 export function createSwapRouter(swapService: SwapService): express.Router {
     const router = express.Router();
     const handlers = new SwapHandlers(swapService);
+    router.get('/', asyncHandler(SwapHandlers.rootAsync.bind(SwapHandlers)));
```

Comment:
confused about what's going on here, should it be `asyncHandler(handlers.rootAsync.bind(handlers)));` ?

Reply:
Nope.